app/views/anonymous_feedback/_feedback.html.erb

```diff
@@ -15,6 +15,13 @@
     <br/>
     <strong>comment</strong>: <%= anonymous_contact.details %>
   <% end %>
+<% elsif anonymous_contact.type == "aggregated-service-feedback" %>
+  <strong><%= anonymous_contact.service_satisfaction_rating %> rating</strong>:
+  <% if anonymous_contact.details.present? %>
+    <%= anonymous_contact.details %>
+  <% end %>
+<% elsif anonymous_contact.type == "service-feedback-rating" %>
```

Comment:
Actually looking at it I think these if statements are the wrong way round? This should be the aggregate, and the above should be the ones with comments?